common/src/main/java/tachyon/resource/ResourcePool.java

```diff
@@ -48,6 +48,20 @@ public ResourcePool(int maxCapacity) {
   }
 
   /**
+   * Test only constructor, used to mock out mResources.
```

Comment:
Since the public constructor should use this one, I would say something along the lines of "Internal constructor that can provide an object to be used for the internal queue".